api-test/src/main/resources/pms/GetListOfAllDevice/GetListOfAllDevice.yml

```
@@ -1,7 +1,7 @@
 GetListOfAllDevice:
    Pms_GetListOfAllDevice_All_Valid_Smoke:
       endPoint: /v1/partnermanager/devicedetail
-      description: Fetching all the active SBI
+      description: Validate that the list of all devices is successfully retrieved with valid inputs
       uniqueIdentifier: TC_PMS_devicedetails_01      
       description: Fetching list of all the devices
```

YAMLlint (1.38.0)

[error] duplication of key "description" in mapping

(key-duplicates)